src/helpers.js

```diff
@@ -119,7 +119,7 @@ export function visitNode(node, index, converters, data, debug) {
   const newProps = Object.assign({}, { key: index }, props);
 
   const children = getChildren(node);
-  const visitChildren = (child, childIndex) => visitNode(child, childIndex, converters, data, debug);
+  const visitChildren = (child, childIdx) => visitNode(child, childIdx, converters, data, debug);
```

Comment:
I had to shorten this line due to eslint rules, that's why I renamed the variable.